cpp/Readme.md

```diff
@@ -0,0 +1,54 @@
+# Starter Code for Bhoos's Smart Bots 2023 
+
+Still wondering why the language that runs the world is missing from this grand competition? <br>
+Now's probably the right time to stop wondering, (>.<) <br>
```

Comment:
@TilakChad We were waiting for chad to show up.

Reply:
xD